src/FastExpressionCompiler/FastExpressionCompiler.cs

```diff
@@ -2641,6 +2738,9 @@ private static void EmitLoadConstantInt(ILGenerator il, int i)
                     case 8:
                         il.Emit(OpCodes.Ldc_I4_8);
                         break;
+                    case -128-127:

```

Comment:
Huh, what is this? a `-255`?

Reply:
oh, you're right, thought it's a range .... my fault...